### PR TITLE
Updating Faucet, NFT Tutorial, and Sending Transaction Document

### PR DESCRIPTION
**Pull request by btandowsky**
I made an update to rinkeby faucet documentation to include links to rinkebyfaucet.com, I edited the NFT tutorial to include the most up to date links (.com instead of .io), and updated a broken link in the send transaction document. Please let me know if you have any questions!

### CODE REVIEW

File: src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ With NFTs bringing blockchain into the public eye, now is an excellent opportuni
 
 Alchemy is extremely proud to be powering the biggest names in the NFT space, including Makersplace (recently set a record digital artwork sale at Christie’s for $69 Million), Dapper Labs (creators of NBA Top Shot & Crypto Kitties), OpenSea (the world’s largest NFT marketplace), Zora, Super Rare, NFTfi, Foundation, Enjin, Origin Protocol, Immutable, and more.
 
-In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [MetaMask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata](https://pinata.cloud/) and [Alchemy](https://dashboard.alchemyapi.io/signup) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
+In this tutorial, we will walk through creating and deploying an ERC-721 smart contract on the Ropsten test network using [MetaMask](https://metamask.io/), [Solidity](https://docs.soliditylang.org/en/v0.8.0/), [Hardhat](https://hardhat.org/), [Pinata](https://pinata.cloud/) and [Alchemy](https://alchemy.com/signup/eth) (don’t fret if you don’t understand what any of this means yet — we will explain it!).
```
Comment (minimalsm) [inline src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:16]: Hey @btandowsky, curious where you got this url? I can't replicate it by navigating through Alchemy's sign up process. The current URL seems to be working fine. Is the /eth/ some sort of tracking? If so, we definitely don't want this.

File: src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Navigate to your [Alchemy Dashboard](https://dashboard.alchemyapi.io/) and creat
 
 ### 2\. Request ETH from the Rinkeby faucet {#request-eth-from-rinkeby-faucet}
 
-Follow the instructions on the [Rinkeby faucet](https://faucet.rinkeby.io/) to receive Eth. You will have to share on social media for this specific faucet. Make sure to include your **Rinkeby** Ethereum address (from Metamask) and not another network. After following the instructions, double check that you’ve received the ETH in your wallet.
+Follow the instructions on the [Rinkeby faucet](https://www.rinkebyfaucet.com/) to receive Eth. You will have to share on social media for this specific faucet. Make sure to include your **Rinkeby** Ethereum address (from Metamask) and not another network. After following the instructions, double check that you’ve received the ETH in your wallet.
```
Comment (minimalsm) [inline src/content/developers/tutorials/sending-transactions-using-web3-and-alchemy/index.md:73]: We should specify this is Alchemy's faucet. It also doesn't require social media validation so removed that.

```suggestion
Follow the instructions on the [Alchemy Rinkeby faucet](https://www.rinkebyfaucet.com/) to receive ETH. Make sure to include your **Rinkeby** Ethereum address (from Metamask) and not another network. After following the instructions, double-check that you’ve received the ETH in your wallet.
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Thanks @btandowsky ❤️

Few comments below.

File: src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md (hunk 1 above)
Comment (btandowsky) [reply src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md:16]: @minimalsm It's a new URL structure that's being introduced, but the current one does work just fine and will be backwards compatible or maintained. Totally aligned with not introducing tracking links! We can reject this update and I'll just ensure we always continue to have functioning URLs in the documentation. Thanks for the help here!

**Comment by btandowsky** [comment]
@minimalsm Let me know if anything else is needed!

**Review by minimalsm — APPROVED** [review]
Thanks for the clarification @btandowsky.

Merging now!

**Pull request merged**